core/src/main/java/org/jdbi/v3/Query.java

```diff
@@ -146,8 +146,8 @@
      * @see Handle#registerRowMapper(RowMapperFactory)
      * @see Handle#registerRowMapper(RowMapper)
      */
-    private <T> Query<T> mapTo(Type resultType) {
-        return this.map(new RegisteredRowMapper<>(resultType, mappingRegistry));
+    public Query mapTo(Type resultType) {
```

Comment:
Hm.... raw type... is the `<T>` really so dangerous that a raw type is better?  I guess it is more truthful about what protections exist...